components/developer/clang-38/Makefile

```diff
@@ -0,0 +1,79 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Aurelien Larcher.  All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= clang-38
+LLVM_NAME=llvm
+CLANG_NAME=cfe
+LIBCXX_NAME=libcxx
```

Comment:
I'm sorry, it's my error initially, but we don't use libcxx for clang, you can remove this line